src/core/Akka/Event/ILoggingAdapter.cs

```diff
@@ -46,10 +44,6 @@ public interface ILoggingAdapter
         /// <param name="args">The arguments.</param>
         void Warning(string format, params object[] args);
 
-        [Obsolete("Use Warning instead!")]
```

Comment:
Probably a good idea to delete `Warn` from the logging adapter interface now - any public interface changes will break the ability to do binding redirects on anything that depends on Akka core and uses this interface, so better off doing that now before stuff starts to pile up.